.eslintignore

```diff
@@ -2,4 +2,5 @@
 dist/
 coverage/
 node_modules/
-package-lock.json
\ No newline at end of file
+package-lock.json
+.eslintrc.json
```

Comment:
This has not been an issue for me in my other projects. I think the Typescript eslint parser I normally use excludes .json files but this project is Javascript and therefore uses a different parser so eslint will try to scan them if they are passed on the command line which lint-staged will do.